docs/Handling_Failed_Nodes.md

```diff
@@ -0,0 +1,72 @@
+# Dealing with Failed Nodes
+
+Nodes can be easily removed from a Clearwater deployment by following the instructions for [elastic scaling](Clearwater_Elastic_Scaling.md). When scaling down the remaining nodes are informed that a node is leaving and take appropriate action. However sometimes a node may fail unexpectedly. If this happens and the node cannot be recovered (for example the virtual machine has been deleted) the remaining nodes must be manually informed of the failure. This article explains how to do this.
+
+## Removing a Failed Node
+
+If a node permanently fails scaling the deployment up and down may stop working, or if a scaling operation is in progress it may get stuck (because other nodes in the tier will wait forever for the failed node to react). To recover from this situation the failed node should be removed from the deployment using the following steps:
+
+* Remove the node from the underlying etcd cluster. To do this:
+    * Run `. /etc/clearwater/config` to access clearwater configuration variables.
+    * Run `etcdctl -C $local_ip:4000 cluster-health` and make a note of the ID of the failed node.
```

Comment:
I wonder if we should have a wrapper script that dots in /etc/clearwater/config and removes the need to specify this. (Alternatively, I don't think it's difficult to make etcd listen on both 127.0.0.1:4001 (the etcdctl default) and $local_ip:4000, but only advertise the latter - @bossmc?)